pandunia-loge.csv

```diff
@@ -2368,7 +2368,7 @@ mas fili||||prefer (favor)|préférer|preferir (favorecer)|preferir|предпо
 mas fili di||||favorite (preferred)|favori (préféré)|favorito (preferido)|favorito (preferido)|любимый|مُفَضَّل||好き (お気に入りの)|||||kegemaran||||suosikki (suosittu)|ulubiony
 mas fili jui||||favoritism|favoritisme|favoritismo|favoritismo|фаворитизм (кумовство)||||||||||||suosinta (favoritismi)|
 mas forte||||reinforcements||refuerzos||пополнение||加固|増派 (援兵)||||||||||wsparcie (wzmocnienie, zasilenie)
-mas ka bas||||too much||demasiado|||||||||||||tro|liikaa|zbyt
+mas ze basa||||too much||demasiado|||||||||||||tro|liikaa|zbyt
```

Comment:
More from enough?

Reply:
I wasn't sure what word would be used for "than", but a lot of languages use a word that also means "from" (there are also many that use "at" or "to").  were you planning to keep a distinct particle for "than"?  I gess **mas da basa** would be another possibility.

Reply:
A distinct particle is probably the best thing.